pkg/controller/pod/pod_controller.go

```diff
@@ -82,7 +82,7 @@ func (r *ReconcilePolicy) Reconcile(request reconcile.Request) (reconcile.Result
 		if errors.IsNotFound(err) {
 			err := r.poolManager.ReleasePodMac(fmt.Sprintf("%s/%s", request.Namespace, request.Name))
 			if err != nil {
-				logging.Printf(logging.ErrorLevel, "failed to release mac for pod %s: %v", request.NamespacedName, err)
+				logging.Errorf("%v, failed to release mac for pod %s: %v",logging.ErrorLevel, request.NamespacedName, err)
```

Comment:
I don't get how is this related to bump of Multus. If it is really needed, please explain it in the commit message.

Reply:
This function so no longer public since this PR https://github.com/intel/multus-cni/pull/339
I updated the commit message.

Reply:
Ack